client/navigation/components/container/style.scss

```diff
@@ -14,8 +14,12 @@
 		height: 100%;
 	}
 
-	.components-navigation__group:nth-of-type(2) {
-		margin-top: auto;
+	.components-navigation__group {
+		margin-bottom: 0;
```

Comment:
The margin I'm seeing is on the item `.components-navigation__item` (as opposed to the group) seen here:

<img width="333" alt="Screen Shot 2020-10-13 at 3 53 23 PM" src="https://user-images.githubusercontent.com/10561050/95909501-986f6b80-0d6c-11eb-8f3b-e62dc37d2bc1.png">

Reply:
Thanks Josh, just checked and it looks like that margin is also being collapsed, so it doesn't extend the spacing there at all.